README.md

```diff
@@ -14,71 +14,68 @@
 
 [![Sauce Test Status](https://saucelabs.com/browser-matrix/js-ipld-resolver.svg)](https://saucelabs.com/u/js-ipld-resolver)
 
-> JavaScript implementation of the IPLD Resolver
+> JavaScript implementation of the IPLD Resolver (internal DAG API module)
 
 ## Table of Contents
 
 - [Install](#install)
 - [Usage](#usage)
 - [API](#api)
   - [IPLD Resolver](#ipldresolver)
-    - [`.put(node, cb)`](#putnode-cb)
-    - [`.putStream([cb])`](#putstreamcb)
-    - [`.get(cid, cb)`](#getcid-cb)
-    - [`.getStream(cid)`](#getstreamcid)
-    - [`.remove(cid, cb)`](#removecid-cb)
+    - [`.put(node, <<cid> || <format>, <hashAlg>>, callback)`](#putnode-cb)
+    - [`.get(cid [, path] [, options], callback)`](#getcid-cb)
+    - [`.remove(cid, callback)`](#removecid-cb)
+    - [`.support.add(multicodec, formatResolver, formatUtil)`]()
+    - [`.support.rm(multicodec)`]()
 - [Contribute](#contribute)
 - [License](#license)
 
 ## Install
 
 ```bash
-> npm install --save ipfs-ipld
+> npm install --save ipld-resolver
 ```
 
 ## Usage
 
 ```js
-const IPLDResolver = require('ipld-resolver')
+const Resolver = require('ipld-resolver')
 
 // pass an optional blockService, if no blockService is passed,
 // one is created in memory.
-const ipldResolver = new IPLDResolver(blockService)
+const Resolver = new Resolver(blockService)
 ```
 
 ## API
 
-### IPLD Resolver
+### `.put(node, <<cid> || <format>, <hashAlg>>, callback)`
```

Comment:
I saw it later on in the code while I was seeing this.
This makes much more sense and it is much cleaner, awesome!

(it just breaks my ipld selector tests! :D)